### ISSUE TITLE
Critical vulnerability in ltijs-sequelize

### ISSUE THREAD

**Issue by slayybelle** [Cvmcosta/ltijs-sequelize#35]
Any way you could patch this critical vulnerability?
No fix available
node_modules/mysql2
  ltijs-sequelize  *
  Depends on vulnerable versions of mysql2
  node_modules/ltijs-sequelize

**Comment by timoconnor** [comment]
Yes can this be fixed?  I wont be allowed to deploy my code if there are critical issues.
